tests/acceptance/test_other_commands.py

```diff
@@ -168,3 +168,11 @@ class TestVersion:
     async def test_version(self, client, node_addresses):
         for node_address in node_addresses:
             assert isinstance(await client.version(node_address), str)
+
+
+class TestStats:
+    async def test_stats(self, client, node_addresses):
+        for node_address in node_addresses:
+            assert type(await client.stats(node_address)) is dict
```

Comment:
Should we also check for the existence of at least one of the stats? otherwise we can not guarantee that [this](https://github.com/emcache/emcache/pull/143/files#diff-d5918e537eadb7095bd9b366b7a19681b87d8d0f87683d5d4007a6fa4d5e861cR797) will be making the work correctly

Reply:
yes, this will be much more correct. The only thing is, let's test established results for memcached and not those in development. I want to do the simplest check, for example `version`